res/values/strings.xml

```diff
@@ -372,7 +372,7 @@
     <string name="upload_move_files">Move file</string>
 
     <string name="pref_behaviour_entries_keep_file">kept in original folder</string>
-    <string name="pref_behaviour_entries_move">moved to app folder</string>
+    <string name="pref_behaviour_entries_move">Moved to app folder</string>
```

Comment:
Settings -> Original file will be...
moved to app folder
kept in original folder